### DISCUSSION TITLE
How to force pre 0.16.1 behavior (no lazy curve precomputation)?

### DISCUSSION THREAD

**Discussion by andrei-datcu** [tlsfuzzer/python-ecdsa#277] [Q&A] [answered]
Is there any way one can opt in for the old behavior where all the curves where computed during import?

Usecase: I'm running some signature checks at certain points in time, each time with a different key and even though this theoretically only impacts the first verification, I'd rather do as much as possible at import time to catch nasty bugs such as this one:
https://github.com/aleaxit/gmpy/issues/317

**Comment by tomato42** [comment, marked as answer]
Any operation with the curve generator will cause the precomputation to happen, so just take a generator for the curve you'll use and then multiply it by 2 or such, like:
```python
from ecdsa import NIST256p
NIST256p.generator * 2
```
iterate over all curves to precompute all
